src/main/java/our/yurivongella/instagramclone/service/PostService.java

```diff
@@ -56,4 +56,18 @@ public Long delete(@NotNull Long postId) {
         postRepository.delete(post);
         return postId;
     }
+
+    public List<PostReadResponseDto> getPostList(Long userId) {
+        Member member = getCurrentMember();
+        List<Post> postList = member.getPosts();
+
+        return postList.stream()
+                .map(post -> PostReadResponseDto.of(post,member))
+                .collect(Collectors.toList());
```

Comment:
`postList` 를 재사용하지 않으면 한줄로도 바꿀수 있습니다. (**피드백 반영 안해도됨**)

```suggestion
        return member.getPosts()
                .stream()
                .map(post -> PostReadResponseDto.of(post,member))
                .collect(Collectors.toList());
```

Reply:
반영하여 수정하였습니다.
https://github.com/INSPJT/instagram_clone_backend/pull/35/commits/8ea65c87236dcc514cf4e4078128f45f20bce485